docs/usage/bazel.md

```diff
@@ -5,22 +5,171 @@ description: Bazel dependencies support in Renovate
 
 # Bazel
 
-Renovate supports upgrading dependencies in Bazel `WORKSPACE` files.
+Renovate upgrades dependencies in Bazel `WORKSPACE` files and `MODULE.bazel` files.
 
 ## How it works
 
 1. Bazel support is enabled automatically
-2. Renovate will search repositories for any `WORKSPACE` files in the repository
-3. Existing dependencies will be extracted from `container_pull`, `oci_pull`, `git_repository`, `go_repository`, `maven_install`, and `http_archive`/`http_file` declarations
-4. Renovate will replace any old versions with the latest version available
+1. Renovate searches the repository for any `WORKSPACE` and `MODULE.bazel` files
+1. Renovate extracts the dependencies it finds from the files (see below for the supported dependency declarations)
+1. Renovate updates old dependencies to the latest version
 
-## git_repository
+## Bazel module (`MODULE.bazel`) support
 
-Renovate will update any `git_repository` declaration that has the following:
+### Bazel registry discovery
 
-1. name
-2. remote matching `https://github.com/<owner>/<repo>.git`
-3. tag using a valid SemVer
+Renovate searches [Bazel registries](https://bazel.build/external/registry) to find new Bazel module versions.
+You customize the registries your Bazel workspace uses by including [`--registry`](https://bazel.build/reference/command-line-reference#flag--registry) entries in your [`.bazelrc` files](https://bazel.build/run/bazelrc).
+Renovate checks the workspace's `.bazelrc` files for custom registry entries.
+If no registries are found, Renovate defaults to the [Bazel Central Registry](https://github.com/bazelbuild/bazel-central-registry).
```

Comment:
The current link points to the GitHub repository for the Bazel Central Registry.

I would expect to land on the registry, based on the link name.

Options:

1. Keep URL to GitHub repository and edit the link name to match the destination
2. Change URL to land on actual registry

```suggestion
If no registries are found, Renovate defaults to the [Bazel Central Registry](https://registry.bazel.build/).
```

I'll let you decide, as you know more about the Bazel users. 😉

Reply:
The actual registry is the GitHub repo. The URL that you suggested is for the catalog. It is a little confusing. I updated the URL to be the one that they advertise in the doc. It is a redirect to the URL that I had.